CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
-# dbt_iterable v0.13.1
+# dbt_iterable v0.14.0
+[PR #59](https://github.com/fivetran/dbt_iterable/pull/59) introduces the following updates:
+
+## Breaking Changes (`--full-refresh` required for Databricks users)
+- Updated `file_format` configuration in `iterable__events` and `int_iterable__list_user_unnest` to be `delta` rather than `parquet`. This resolves Databricks incompatible configuration errors that arise with the parquet format. 
```

Comment:
Since this is kinda a schema change for Databricks users, we could put this in the Schema/Data Changes table format. Not a hard requirement tho; I'll leave it up to you to decide